packages/docusaurus-module-type-aliases/src/index.d.ts

```diff
@@ -319,6 +319,17 @@ declare module '@docusaurus/useGlobalData' {
   export default useGlobalData;
 }
 
+declare module '*.svg' {
+  import type {FunctionComponent, SVGProps} from 'react';
+
+  export const ReactComponent: FunctionComponent<
+    SVGProps<SVGSVGElement> & {title?: string}
+  >;
+
+  const src: string;
+  export default src;
```

Comment:
Are you sure this is correct...? Afaik since we use the `svgr` loader, the default export is already the inline component: https://docusaurus.io/docs/markdown-features/assets#inline-svgs

Reply:
Corrected in 6c677b9.